.hintrc

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
![Capture1](https://user-images.githubusercontent.com/93039359/210941732-9afc0829-d766-4423-92d0-d2978da7fc30.PNG)
- [ ] Your link to your license file is not working becuase you didn't create any link :fearful: Kindly add a license file using these [guildlines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) :blush: :+1:

- [ ] After making that license file kindly link that license file in your readme file :thinking:

Reply:
thanks, done